api/src/main/resources/liquibase.xml

```diff
@@ -1543,11 +1531,26 @@
         </insert>
     </changeSet>
 
-    <changeSet id="muzima-20180720102200" author="bmokaya">
+    <changeSet id="muzima-20190807172200" author="pwangoo">
+        <preConditions onFail="MARK_RAN">
+            <and>
+                <columnExists tableName="muzima_notification_data" columnName="patient"/>
+            </and>
+            <sqlCheck expectedResult="NO">
+                SELECT is_Nullable
+                FROM  INFORMATION_SCHEMA.COLUMNS
+                WHERE table_name='muzima_notification_data'
+                AND column_name='patient'
+            </sqlCheck>
+        </preConditions>
         <dropNotNullConstraint columnDataType="int" columnName="patient" tableName="muzima_notification_data"/>
     </changeSet>
 
-    <changeSet id="muzima-20180717115600" author="mssavai">
+    <changeSet id="muzima-20190813265900" author="pwangoo">
```

Comment:
Why the change of changeset id and author. Please also consider all the changes below this comment.